antora-ui-camel/src/js/vendor/algoliasearch.bundle.js

```diff
@@ -25,17 +28,23 @@
       }
     }
 
+    search.addEventListener('click', function (e) {
+      e.stopPropagation()
+    })
+
     search.addEventListener(
       'keyup',
       debounce((key) => {
+        if (condition) document.querySelector('.navbar-burger').style.marginTop = '1rem'
```

Comment:
I'm not sure that the content security policy will allow changing the style via JavaScript, anyhow we need to minimize the JavaScript and focus on style changes via CSS only.

Reply:
Yes, I experimented and it is possible through HTML and CSS only. I will push the changes for the same.